paymentsheet/src/main/java/com/stripe/android/lpmfoundations/paymentmethod/definitions/CardDefinition.kt

```diff
@@ -150,7 +150,10 @@ private fun contactInformationElement(
         ).takeIf { collectEmail },
         PhoneNumberElement(
             identifier = IdentifierSpec.Phone,
-            controller = PhoneNumberController(initialValues[IdentifierSpec.Phone] ?: "")
+            controller = PhoneNumberController.createPhoneNumberController(
+                initialValue = initialValues[IdentifierSpec.Phone] ?: "",
+                initiallySelectedCountryCode = initialValues[IdentifierSpec.Country]
```

Comment:
I think this will break initially entered phone numbers that may have a different country than the country of the billing address.
https://github.com/stripe/stripe-android/blob/1a68b6c77a660674415752ac4c0bc9f1498a0c1d/stripe-ui-core/src/main/java/com/stripe/android/uicore/elements/PhoneNumberController.kt#L130-L147

Reply:
Could test this by making the default phone number in the `PaymentSheetPlayground` a UK phone number.

Reply:
thanks for catching this! There were a few places where I had added passing the initial country. Reverted all of those so that this bug also doesn't show up anywhere else